src/js/other-websites/inpage_menu.js

```diff
@@ -311,32 +323,35 @@ function hasMaskBeenUsedOnCurrentSite(mask, domain) {
   const domainList = mask.used_on;
 
   // Short circuit out if there's no used_on entry
-  if (domainList === null || domainList === "" ||  domainList === undefined) { return false; }
+  if (domainList === null || domainList === "" || domainList === undefined) {
+    return false;
+  }
 
   // Domain already exists in used_on field. Just return the list!
   if (domainList.includes(domain)) {
     return true;
   }
 
-  // No match found! 
+  // No match found!
   return false;
 }
 
 function haveMasksBeenUsedOnCurrentSite(masks, domain) {
-  return masks.some(mask => {
-    
+  return masks.some((mask) => {
     const domainList = mask.used_on;
 
     // Short circuit out if there's no used_on entry
-    if (domainList === null || domainList === "" ||  domainList === undefined) { return false; }
+    if (domainList === null || domainList === "" || domainList === undefined) {
```

Comment:
suggestion (non-blocking): I've seen this logic at least a handful of times - I wonder if we should have a function that does this check and reuse it.